streampipes-connect-adapters/src/main/java/org/apache/streampipes/connect/protocol/set/FileProtocol.java

```diff
@@ -79,11 +84,32 @@ public ProtocolDescription declareModel() {
     @Override
     public Protocol getInstance(ProtocolDescription protocolDescription, Parser parser, Format format) {
         StaticPropertyExtractor extractor = StaticPropertyExtractor.from(protocolDescription.getConfig());
-
+        this.timeBetweenReplay = extractor.singleValueParameter(INTERVAL_KEY, Integer.class);
         String fileFetchUrl = extractor.selectedFileFetchUrl("filePath");
         return new FileProtocol(parser, format, fileFetchUrl);
```

Comment:
```suggestion
        return new FileProtocol(parser, format, fileFetchUrl, timeBetweenReplay);
```
Please adapt the constructor accordingly